src/components/share-overlay/_share-overlay.scss

```diff
@@ -27,6 +29,12 @@
 
       .input-copy-url {
         margin: 0;
+
+        .form-control {
+          background-color: $tone-8-color;
```

Comment:
same question as in downloads - this means that share will now be affected by changes in color variables - is that what product want ?

Reply:
Right